connectors/sources/servicenow.py

```diff
@@ -369,6 +400,65 @@ async def close_session(self):
         del self._get_session
 
 
+class ServiceNowAdvancedRulesValidator(AdvancedRulesValidator):
+    RULES_OBJECT_SCHEMA_DEFINITION = {
+        "type": "object",
+        "properties": {
+            "service": {"type": "string", "minLength": 1},
```

Comment:
Can `service` be an arbitrary string or must it be a specific element of a specific set of strings?

Reply:
Only a `set of alphabets` is acceptable for valid service.